assets/numbat.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Numbat
+GenericName=Scientific calculator
```

Comment:
Suggestions for a different generic name maybe?

Reply:
Maybe check what similar applications use?

Reply:
Qalculate uses just "Calculator," we could go with that?